Jenkinsfile

```diff
@@ -906,8 +921,7 @@ pipeline {
                             }
                         }
                         unitTest timeout_time: 60,
-                                 inst_repos: el7_component_repos + ' ' +
-                                             component_repos(),
+                                 inst_repos: daos_repos(),
```

Comment:
This appears to now be installing: https://github.com/daos-stack/daos/blob/5b3d28bcf2005ddaecfd1b61c37ccc9820179d67/Jenkinsfile#L88 which is `daos_repo()` more than it was installing before, isn't it?